src/components/Chart.jsx

```diff
@@ -21,143 +21,129 @@ import { connect } from '../store/Connect';
-                        'smartcharts-desktop': !isMobile,
-                    })}
-                >
-                    <div className='cq-context' ref={this.root}>
-                        <div className={currentPosition}>
-                            <div className='ciq-chart-area'>
-                                <div className={classNames('ciq-chart', { 'closed-chart': isChartClosed })}>
-                                    <RenderInsideChart at='holder'>
-                                        {barriers.map((barr, idx) => (
-                                            <Barrier
-                                                key={`barrier-${idx}`} // eslint-disable-line react/no-array-index-key
-                                                {...barr}
-                                            />
-                                        ))}
-                                    </RenderInsideChart>
-                                    <RenderInsideChart at='subholder'>
-                                        {!isCandle && !isSpline && isHighestLowestMarkerEnabled && (
-                                            <HighestLowestMarker />
-                                        )}
-                                    </RenderInsideChart>
-                                    <RenderInsideChart at='subholder' hideInScrollToEpoch>
-                                        {children}
-                                    </RenderInsideChart>
-                                    <RenderInsideChart at='subholder'>
-                                        <PaginationLoader />
-                                    </RenderInsideChart>
-                                    <div className='cq-top-ui-widgets'>
-                                        <TopWidgets />
-                                    </div>
-                                    <div className='chartContainer' style={{ height: chartContainerHeight }}>
-                                        <Crosshair />
-                                    </div>
-                                    {enabledNavigationWidget && (
-                                        <NavigationWidget onCrosshairChange={onCrosshairChange} />
-                                    )}
-                                    {toolbarWidget && <ToolbarWidget />}
-                                    {!isChartAvailable && (
-                                        <div className='cq-chart-unavailable'>
-                                            {t.translate('Chart data is not available for this symbol.')}
-                                        </div>
-                                    )}
-                                    <BottomWidgetsContainer>
-                                        <BottomWidget bottomWidgets={bottomWidgets} />
-                                    </BottomWidgetsContainer>
+                <div className='cq-context' ref={rootRef}>
+                    <div className={currentPosition}>
```

Comment:
@balakrishna-binary can we use `classNames` lib here and remove `currentPosition`?

Reply:
done 😄